lib/NAnt/NAnt.exe.config

```diff
@@ -3067,7 +3075,7 @@
                         </if>
                         <property name="resgen.supportsexternalfilereferences" value="false" />
                         <property name="prefix" value="${pkg-config::get-variable('mono', 'prefix')}" />
-                        <property name="toolDirectory" value="${path::combine(prefix, 'lib/mono/4.0')}" />
+                        <property name="toolDirectory" value="${path::combine(prefix, 'lib/mono/4.5')}" />
                     </project>
                     <tasks>
                         <task name="al">
```

Comment:
Typically in this file you simply add ANOTHER framework, so the old one is still here.

Reply:
Commit pushed with mono-4.0 framework added in addition to mono-4.5.